src/utils/enforce-array-arguments.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import invariant from 'invariant'
+
+const generateInvalidFunctionError = (func) => (
+`Error: ${func} is called as a function.
```

Comment:
I think this first line is not necessary, especially since it will prepend "Invariant Violation:" there, if I'm not mistaken?

```js
error.name = 'Invariant Violation';
```